pkg/numbers/number.go

```diff
@@ -71,12 +71,12 @@ func SliceAtoi(sa []string) ([]int, error) {
 	return si, nil
 }
 
-func GetInterval(blockTime int, minInterval, maxInterval time.Duration) (pollInterval time.Duration) {
-	pollInterval = time.Duration(blockTime) * time.Millisecond
-	if pollInterval < minInterval {
-		pollInterval = minInterval
-	} else if pollInterval > maxInterval {
-		pollInterval = maxInterval
+func GetInterval(value int, minInterval, maxInterval time.Duration) (interval time.Duration) {
+	interval = time.Duration(value) * time.Millisecond
+	if interval < minInterval {
+		interval = minInterval
```

Comment:
did you consider using `max`, `min` functions for this? would be much more readable

Reply:
yes! but we are comparing time. interval, if we use the Max or Min function we need to do a lot of casting

Reply:
There is two pieces:
1) calculate internal (observer specific)
2) get min internal (just min function? why complicated with such design?)

I do not understand why it needs a separate function in the numbers package?

Reply:
please, give me your suggestion, because it's not clear whats you are trying to say and we can spend less time fixing a simple thing. It's good for me this way, works, is testable, clean and  I don't wanna put this logic inside the observer's main package (cmd). 

The max and min function is used for `int`, we are comparing `time.Duration`

Reply:
https://en.wikipedia.org/wiki/Single_responsibility_principle